MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/MuonFitPropertyBrowser.h

```diff
@@ -72,6 +72,12 @@ class EXPORT_OPT_MANTIDQT_MANTIDWIDGETS MuonFitPropertyBrowser
   std::string outputName() const override;
   /// Prefix for simultaneous fit results
   static const std::string SIMULTANEOUS_PREFIX;
+  /// custom label
+  static const std::string CUSTOM_LABEL;
```

Comment:
If this is the only file these are used in can we just move them into an anonymous namespace in the .cpp file instead of making them visible at a larger namespace